### PR TITLE
Add i18n module

### PR DESCRIPTION
**Pull request by chimp1984**

### CODE REVIEW

File: i18n/src/main/java/network/misq/i18n/UTF8Control.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.charset.StandardCharsets;
+import java.util.Locale;
+import java.util.PropertyResourceBundle;
+import java.util.ResourceBundle;
+
+// Adds UTF8 support for property files
```
Comment (kaiwolfram) [inline i18n/src/main/java/network/misq/i18n/UTF8Control.java]: If you use JavaDoc comments your IDE can display it in the documentation popup
```suggestion
/**
 * Adds UTF8 support for property files
 */
```

File: i18n/src/main/java/network/misq/i18n/UTF8Control.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        } else {
+            stream = loader.getResourceAsStream(resourceName);
+        }
+        if (stream != null) {
+            try {
+                // Only this line is changed to make it read properties files as UTF-8.
+                bundle = new PropertyResourceBundle(new InputStreamReader(stream, StandardCharsets.UTF_8));
+            } finally {
+                stream.close();
+            }
```
Comment (kaiwolfram) [inline i18n/src/main/java/network/misq/i18n/UTF8Control.java]: try-with-resources automatically closes the resource. Not 100% sure if you can use an already declared one tho
```suggestion
            try (stream){
                // Only this line is changed to make it read properties files as UTF-8.
                bundle = new PropertyResourceBundle(new InputStreamReader(stream, StandardCharsets.UTF_8));
            } 
```

Comment (chimp1984) [reply i18n/src/main/java/network/misq/i18n/UTF8Control.java]: Yes agree would be better to use try-with-resources. Just pushed a commit

**Review by jmacxx — APPROVED** [review]
Open question about whether git whitespace warnings are acceptable.
Other than that minor issue, ACK.

**Comment by chimp1984** [comment]
> Open question about whether git whitespace warnings are acceptable.

Where do you get that?

File: buildSrc/lombok-dependencies.gradle [hunk 3]
```diff
@@ -0,0 +1,7 @@
+dependencies {
+    api platform(project(':platforms:common-platform'))
```
Comment (ghubstan) [inline buildSrc/lombok-dependencies.gradle]: Commit 6edd97ce5cf3320f676525253bcf615e96c91644 removes `api platform(project(':platforms:common-platform'))` as per the commit's comment.

**Comment by ghubstan** [comment]
> > Open question about whether git whitespace warnings are acceptable.
> 
> Where do you get that?

It is odd, new git diff behaviour.  If I add a newline to the end of a file, git complains:
```
$ git diff buildSrc/lombok-dependencies.gradle
diff --git a/buildSrc/lombok-dependencies.gradle b/buildSrc/lombok-dependencies.gradle
index cac2e6a..e968399 100644
--- a/buildSrc/lombok-dependencies.gradle
+++ b/buildSrc/lombok-dependencies.gradle
@@ -5,4 +5,4 @@ dependencies {
     annotationProcessor 'org.projectlombok:lombok:1.18.20'
     testCompileOnly 'org.projectlombok:lombok:1.18.20'
     testAnnotationProcessor 'org.projectlombok:lombok:1.18.20'
-}
\ No newline at end of file
+}

```

File: i18n/src/main/resources/logback.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ghubstan) [inline i18n/src/main/resources/logback.xml]: Commit 5db7a1ac952deaf80676dab9bc7dffd33a9d1e91 removes i18n/src/main/resources/logback.xml.  See the commit's comment.

**Comment by ghubstan** [comment]
I made two changes (6edd97ce5cf3320f676525253bcf615e96c91644, 5db7a1ac952deaf80676dab9bc7dffd33a9d1e91) I cannot review myself.  This PR can be merged if @jmacxx  reivews and ACKs it again.

**Review by jmacxx — APPROVED** [review]
utACK

**Comment by ghubstan** [comment]
ACK

**Pull request merged**